### PR TITLE
[ACM-6607]: Indicate that hosted clusters are automatically imported

### PR DESCRIPTION
**Pull request by lahinson**
Related Jiras: 
- https://issues.redhat.com/browse/ACM-6607 (main Jira)
- https://issues.redhat.com/browse/ACM-6858 (related to adding docs about disabling auto import)

### CODE REVIEW

**Comment by dtthuynh** [comment]
Thanks @lahinson 
Overall looks good, I know this is separate from your changes here, but I believe we do not need line 15-31 anymore (@rokej can correct me if I'm misspeaking):
image

**Comment by lahinson** [comment]
Thanks @dtthuynh! I'll wait for Roke's response before I remove those lines.

**Comment by rokej** [comment]
@dtthuynh @lahinson That is correct that we don't need lines 15 ~ 31 anymore. Thank you!

File: clusters/hosted_control_planes/importing_hosted_cluster_aws.adoc [hunk 1]
```diff
@@ -1,37 +1,16 @@
 [#importing-hosted-cluster-aws]
 = Importing a hosted control plane cluster on AWS
 
-You can import a hosted control plane cluster with the console.
+Hosted clusters are automatically imported into {mce-short} after the hosted control plane becomes available. The following steps describe how to import hosted clusters manually.
```
Comment (oafischer) [inline clusters/hosted_control_planes/importing_hosted_cluster_aws.adoc]:
```suggestion
Hosted clusters are automatically imported into {mce-short} after the hosted control plane becomes available. The following steps describe how to import hosted clusters manually:
```

**Review by oafischer — APPROVED** [review]
Looks great!

File: clusters/hosted_control_planes/importing_hosted_cluster_aws.adoc (hunk 1 above)
Comment (dockerymick) [inline clusters/hosted_control_planes/importing_hosted_cluster_aws.adoc]: My main change request is to add the colon after "manually", but here is my complete suggestion: 

```suggestion
Hosted clusters are automatically imported into {mce-short} after the hosted control plane becomes available. If you want to import hosted clusters manually, complete the following steps:
```

**Review by dockerymick — APPROVED** [review]
Left a suggestion and checked the links. Otherwise, this looks good to me! Thanks !

File: clusters/hosted_control_planes/importing_hosted_cluster_aws.adoc (hunk 1 above)
Comment (lahinson) [reply clusters/hosted_control_planes/importing_hosted_cluster_aws.adoc]: Great suggestion!

**Comment by lahinson** [comment]
Thanks @dockerymick and @oafischer! I incorporated both of your suggestions.

**Pull request merged**